src/Rocksmith2014.Common/FSharpJsonOptions.fs

```diff
@@ -7,6 +7,6 @@ type FSharpJsonOptions() =
     static member Create(?indent, ?ignoreNull) =
         JsonSerializerOptions(
             WriteIndented = defaultArg indent false,
-            IgnoreNullValues = defaultArg ignoreNull false
+            DefaultIgnoreCondition = defaultArg ignoreNull JsonIgnoreCondition.Never
```

Comment:
I'd rather keep the `ignoreNull` parameter as a boolean.
```fsharp
        let ignoreNull = defaultArg ignoreNull false

        JsonSerializerOptions(
            WriteIndented = defaultArg indent false,
            DefaultIgnoreCondition = if ignoreNull then JsonIgnoreCondition.WhenWritingNull else JsonIgnoreCondition.Never
        )
```
That way fewer changes are needed. There were also still some call sites that were not changed.

Reply:
That definitely makes more sense.

And yeah, I just realized I missed rebuilding your solution before pushing (I made the changes in my solution, which only references a subset of the Rocksmith2014.NET projects, so only that subset had call sites updated)